tool/jt.rb

```diff
@@ -1701,33 +1701,37 @@ def install(name)
   def install_jvmci
     raise "Installing JVMCI is only available on Linux and macOS currently" unless LINUX || MAC
 
+    jvmci_version = ENV["JVMCI_VERSION"] || Utilities.jvmci_version
     dir = File.expand_path("..", TRUFFLERUBY_DIR)
     java_home = chdir(dir) do
       if LINUX
-        jvmci_version = ENV["JVMCI_VERSION"] || Utilities.jvmci_version
-        jvmci_grep = "#{dir}/openjdk1.8.0*#{jvmci_version}"
-        if Dir[jvmci_grep].empty?
+        dir_grep = "#{dir}/openjdk1.8.0*#{jvmci_version}"
```

Comment:
I would name this `pattern` or so.

Reply:
(it's a glob pattern, not a grep pattern)

Reply:
Fixed.